### PR TITLE
Add Address component

### PR DESCRIPTION
**Pull request by EmilMalanczak**

### CODE REVIEW

File: packages/app/src/features/dialogs/savings/withdraw/views/SuccessView.tsx [hunk 1]
```diff
@@ -40,19 +40,24 @@ export function SuccessView({ tokenToWithdraw, closeDialog, sendModeExtension }:
...
-                    className="flex items-center gap-2.5 font-medium text-basics-dark-grey text-sm"
-                  >
-                    <BoxArrowTopRight className="h-4 w-4" />
-                  </Link>
-                )}
+              <Typography variant="prompt" className="flex min-w-0 flex-1 gap-1.5 text-sm">
+                <Address
+                  className="justify-end"
+                  address={sendModeExtension.receiver}
+                  inlineIcon={
```
Comment (oskarvu) [inline packages/app/src/features/dialogs/savings/withdraw/views/SuccessView.tsx:49]: postfixIcon

File: packages/app/src/features/dialogs/savings/withdraw/views/SuccessView.tsx [hunk 2]
```diff
@@ -40,19 +40,24 @@ export function SuccessView({ tokenToWithdraw, closeDialog, sendModeExtension }:
...
-                  <Link
-                    to={sendModeExtension.blockExplorerAddressLink}
-                    external
-                    className="flex items-center gap-2.5 font-medium text-basics-dark-grey text-sm"
-                  >
-                    <BoxArrowTopRight className="h-4 w-4" />
-                  </Link>
-                )}
+              <Typography variant="prompt" className="flex min-w-0 gap-1.5 text-sm">
+                <Address
```
Comment (oskarvu) [inline packages/app/src/features/dialogs/savings/withdraw/views/SuccessView.tsx:46]: I guess here you can use `BlockExplorerAddressLink` you created. Then maybe modify `sendModeExtension` so it doesn't return link address and user `receiver` here.

File: packages/app/src/ui/utils/useResizeObserver.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  ref: RefObject<T>
+  onResize?: (size: Size) => void
+}
+
+const initialSize: Size = {
+  width: 0,
+  height: 0,
+}
+
+// Base on https://usehooks-ts.com/react-hook/use-resize-observer
```
Comment (krzkaczor) [inline packages/app/src/ui/utils/useResizeObserver.ts:20]:
```suggestion
// Based on https://usehooks-ts.com/react-hook/use-resize-observer
```

File: packages/app/src/features/dialogs/savings/withdraw/views/SuccessView.tsx (hunk 2 above)
Comment (oskarvu) [inline packages/app/src/features/dialogs/savings/withdraw/views/SuccessView.tsx:46]: I guess here you can use `BlockExplorerAddressLink` you created. Then maybe modify `sendModeExtension` so it doesn't return link address and just use `receiver` here.

**Review by oskarvu — APPROVED** [review]

**Review by krzkaczor — APPROVED** [review]

**Review by yivlad — APPROVED** [review]

**Pull request merged**